ruby/rb-nokogiri/Portfile

```diff
@@ -4,7 +4,10 @@ PortSystem          1.0
 PortGroup           ruby 1.0
 
 ruby.branches       2.7 2.6 2.5 2.4
-ruby.setup          nokogiri 1.10.4 gem
+# Return to stable version after upgrading to 1.11.0. The stable version will \
+# fail if it does not find mini_portile2 version 2.4.0.
+# See https://github.com/macports/macports-ports/pull/9116
```

Comment:
Why not stay with the stable version and just add a patch:

https://github.com/sparklemotion/nokogiri/pull/2005/files

Reply:
I wanted to avoid building from source as I would have to add a lot of new dependencies-packages that MacPorts does not have. But I will try this.

Reply:
1.11.0 has now been releeased. 1.11.1 too. https://nokogiri.org/CHANGELOG.html